src/commands/run.py

```diff
@@ -113,3 +123,22 @@ def remove_done_tasks():
             session.commit()
             Session.remove()
             run_print('Done')
+
+    def get_confirmation(configs, nodes):
+        click.echo("""
+You are about to run: {}\nOver nodes: {}\n
+""".strip().format(', '.join([c.name() for c in configs]),
+                   ', '.join(nodes)))
+        question = "Please enter [y/n] or 'more' for more info"
+        while "answer is invalid":
+            reply = click.prompt(question)
+            if reply == 'y':
+                return True
+            if reply == 'n':
+                return False
+            if reply == 'more':
```

Comment:
Do we need the `more` options? Has it been requested?

Reply:
I only added it as it was no effort & could imagine a large list of help commands